homework_11/homework_11.py

```diff
@@ -0,0 +1,78 @@
+# # 1 Написати власний декоратор, задачею якого має бути друк назви функції і
+# # часу, коли вона була викликана. Декоратор має працювати для різних функцій однаково.
+# import time
+
+
+# def deco_time_and_name(func_to_deco):
+#     def print_time_and_name():
```

Comment:
Use *args and **kwargs here to make it works with functions that take arguments. And pass these arguments to the `func_to_deco` call